docs/how-to/installation/kubernetes.txt

```diff
@@ -178,6 +180,15 @@ GCS:
    for S3 clones (e.g.,
    ``http://<minio-endpoint>:<minio-port|default=9000>``).
 
+-  **Microsoft Azure Blob Storage**: To configure Determined to use Microsoft Azure Blob Storage for checkpoint and TensorBoard storage, users need to set ``checkpointStorage.type`` in ``values.yaml`` to ``azure``
```

Comment:
sometimes `rstfmt` works in mysterious ways?.. this is obviously over 80 character limit.